### DISCUSSION TITLE
DgnCustomItemTypes appears in different place in Schema Explorer

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3396] [Q&A] [answered]
Today I've created new item types in MicroStation. When explorering iModel Schema Explorer, I find that the newly created item types is under PyhsicalElement.
![image](https://user-images.githubusercontent.com/19658624/159399275-4dfb1c14-ea12-4cbc-85d7-80799fd3bc11.png)
But a weeks ago, It was under ElementMultiAspect. (I didn't capture a picture last week, but found a similar one throught others blog).
![image](https://user-images.githubusercontent.com/19658624/159399297-a40f5908-ac83-4239-9ff6-4d84206524f1.png)
Is this an bug of iTwin Synchronizer or a update?

**Comment by pmconne** [comment, marked as answer]
Hi @Snailya,

Did you convert the exact same DGN file with the exact same content both times?
When you say "today I've createtd new item types" it suggests a different DGN file.
In each case, was the "use item name for element" checkbox enabled for the Item Type(s)? That option influences whether the Item becomes an element or an aspect.
